src/routes/safe/components/Transactions/TxList/modals/ApproveTxModal.tsx

```diff
@@ -190,21 +190,31 @@ type Props = {
 export const ApproveTxModal = ({ onClose, isOpen, transaction }: Props): React.ReactElement => {
   const dispatch = useDispatch()
   const userAddress = useSelector(userAccountSelector)
+  const isOwner = useSelector(grantedSelector)
   const classes = useStyles()
   const safeAddress = extractSafeAddress()
+  const txInfo = useTxInfo(transaction)
+
   const executionInfo = transaction.executionInfo as MultisigExecutionInfo
-  const thresholdReached = !!(executionInfo && isThresholdReached(executionInfo))
+  const { confirmationsSubmitted, confirmationsRequired } = executionInfo
```

Comment:
I know it's legacy but we should probably use the type check function here to be safe.

Reply:
Why did you remove the helper function as well? It's only used here?

Reply:
It was only used here and was just a `>=-comparison`